packages/blockchain-info-components/src/Form/AutosizeTextInput.js

```diff
@@ -72,7 +72,17 @@ class TextInput extends React.Component {
   }
 
   render () {
-    const { errorState, disabled, noLastPass, ...rest } = this.props
+    const {
+      errorState,
+      disabled,
+      noLastPass,
+      borderColor,
+      focusedBorderColor,
+      borderRightNone,
+      minHeight,
+      active,
+      ...rest
+    } = this.props
```

Comment:
this fix console errors based on passing along non DOM properties

Reply:
👍